web/src/main/java/org/hillview/targets/InitialObjectTarget.java

```diff
@@ -88,6 +93,8 @@ public InitialObjectTarget() {
 
     private void initialize(final HostList description) {
         this.emptyDataset = RemoteDataSet.createCluster(description, RemoteDataSet.defaultDatasetIndex);
+        PingSketch<Empty> ping = new PingSketch<Empty>();
+        this.workers = new ArrayList<>(Objects.requireNonNull(this.emptyDataset.blockingSketch(ping)));
```

Comment:
we have our own function checkNonNull. Perhaps we shouldn't have, but let's use it.

Reply:
I think it's better to use the one provided by java, but I have changed to our checkNull function for now